src/main/resources/reference.conf

```diff
@@ -0,0 +1,4 @@
+jicofo {
+  # The region in which the machine is running.
```

Comment:
I was thinking about a convention where comments in HOCON files use `//` and commented-out properties (for the purpose of examples or whatever) use `#`, to help differentiate them.  Thoughts?

Reply:
(I did this in Jibri and found that it helped readability, in my opinion)